### ISSUE TITLE
Visual Studio 2022 support

### ISSUE THREAD

**Issue by vsopko** [Wakusei/BuildStartProject#3]
Subj

**Comment by Wakusei** [comment]
Yeah, I'm going to support it. I already changed the source code and now waiting for the release of VS.

**Comment by Wakusei** [comment]
I updated the project settings including the SDK and now it's not working. The build command in VS is not responding. I'm investigating it.

**Comment by Wakusei** [comment]
I removed experimental code which I forgot to delete. Now VS2022 is supported and the exntesion has been published.
